.travis.yml

```diff
@@ -19,6 +19,10 @@ before_install:
 
 install:
   - yarn install
+  - npm install -g ethereumjs-testrpc
```

Comment:
should we use either npm or yarn solely for package management

Reply:
Ah I was not aware that yarn can install every npm package. Will change.